cmd/agent/app/flare.go

```diff
@@ -116,42 +137,75 @@ func makeFlare(caseID string) error {
 	return nil
 }
 
-func requestArchive(logFile string) (string, error) {
+func requestArchive(logFile string, profile *flare.Profile) (string, error) {
 	fmt.Fprintln(color.Output, color.BlueString("Asking the agent to build the flare archive."))
```

Comment:
IMHO these "codecov" checks are making reviews quite difficult and are only helpful to the author. They could easily remain as part of the CI.